lib/matplotlib/table.py

```diff
@@ -211,12 +242,15 @@ def __init__(self, ax, loc=None, bbox=None, **kwargs):
         self.set_clip_on(False)
 
         self._cachedRenderer = None
+        self._cellType = 'default'
+        self._cellCreation = {"default": Cell, "scicell": SciCell}
```

Comment:
This is an instance level attribute.  A class level would look something like

```python
class foo(object):
    bar = {'a': 1, 'b': 2}
    def __init__(self, lab):
        if lab not in self.bar:
            raise ValueError("invalid key")
       self.label = self.bar[lab]
```

The `bar` attribute is shared between all instances of the class so changing it results in it changing for _all_ instances.  This can be useful if you want to set up a registry of available `Cell` types.